### PR TITLE
[move-book and move-book-zh] update summary and remove english contents that has been proof

### PR DESCRIPTION
**Pull request by geometryolife**
## Motivation

Update summary file of move-book-zh and fix some typos.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/move-language/move/blob/main/CONTRIBUTING.md#developer-workflow)?

Yes

### CODE REVIEW

**Review by jiangying000 — APPROVED** [review]

File: language/documentation/book/translations/move-book-zh/src/signer.md [hunk 1]
```diff
@@ -77,16 +53,8 @@ The `std::signer` standard library module provides two utility functions over `s
...
 此外，`move_to<T>(&signer, T)` [全局存储](./global-storage-operators.md)操作符需要一个 `&signer` 参数在 `signer.address` 的帐户下发布资源 `T`。这确保了只有经过身份验证的用户才能在其地址下发布资源。
 
-## 所有权（Ownership）
-
-Unlike simple scalar values, `signer` values are not copyable, meaning they cannot be copied (from
-any operation whether it be through an explicit [`copy`](./variables.md#move-and-copy) instruction
-or through a [dereference `*`](./references.md#reference-operators)).
+## 所有权
 
 与简单的标量值不同，`signer` 值是不可复制的，这意味着他们不能被复制（通过任何操作，无论是通过显式 [`copy`](./variables.md#移动和复制move-and-copy)指令还是通过[解引用（dereference）`*`](./references.md#reference-operators)）。
```
Comment (jiangying000) [inline language/documentation/book/translations/move-book-zh/src/signer.md:60]: would be better linking [dereference `*`]/[解引用（dereference）`*`] to https://move-language.github.io/move/references.html#reading-and-writing-through-references

File: language/documentation/book/translations/move-book-zh/src/vector.md [hunk 2]
```diff
@@ -177,28 +136,17 @@ fun destroy_droppable_vector<T: drop>(vec: vector<T>) {
...
-For more details see the sections on [type abilities](./abilities.md) and [generics](./generics.md).
-
 大向量的复制可能很昂贵，因此编译器需要显式 `copy` 以便更容易查看它们发生的位置。
 
 有关更多详细信息，请参阅[类型能力](./abilities.md)和[泛型](./generics.md)部分。
 
-## 所有权（Ownership）
+## 所有权
 
 [如上所述](#销毁和复制-vectordestroying-and-copying-vector)，`vector` 值只有在元素值可以复制的时候才能复制。在这种情况下，复制必须通过显式 [`copy`](./variables.md#移动和复制move-and-copy) 或者[解引用 `*`](./references.md#引用运算符reference-operators)。
```
Comment (jiangying000) [inline language/documentation/book/translations/move-book-zh/src/vector.md:152]: [如上所述](#销毁和复制-vectordestroying-and-copying-vector), this link doesn't work

Comment (geometryolife) [reply language/documentation/book/translations/move-book-zh/src/vector.md:152]: > `[如上所述](#销毁和复制-vectordestroying-and-copying-vector)`, this link doesn't work

Yes, I haven't updated the internal index of content, I'll update it now.

**Review by jiangying000 — APPROVED** [review]

File: language/documentation/book/translations/move-book-zh/src/signer.md (hunk 1 above)
Comment (jiangying000) [reply language/documentation/book/translations/move-book-zh/src/signer.md]: 感觉可以把 move book 英文版的这个链接也顺便修了

**Review by sblackshear — APPROVED** [review]

**Pull request merged**